src/workerd/jsg/jsg.h

```diff
@@ -1970,6 +1971,12 @@ class Lock {
   using Logger = void(Lock&, kj::StringPtr);
   void setLoggerCallback(kj::Function<Logger>&& logger);
 
+  // ---------------------------------------------------------------------------
+  // Misc. Stuff
+
+  void requestGcForTesting() const;
+  // Sends an Immediate request for full GC
```

Comment:
If we are going to put this on `jsg::Lock`, let's extend this comment to stress very strongly that this is to be used *ONLY* in tests. In fact, we may want to make it throw if not running in predictable mode.

Reply:
Done, it will fatally throw if the function is called outside of testing.